### ISSUE TITLE
Plantagenet Cherokee font is squished in the newer version of Word

### ISSUE THREAD

**Issue by cwyTommylee** [MicrosoftDocs/typography-issues#775]
I've been using the Plantagenet Cherokee for studying and typing in Cherokee since the font was introduced. I corresponded the designer of the font not long after the release and discussed some issues with him at that time. What happened to it??  Now, with the new version of MS Word, version 16 for Mac, when I open my old documents, the top half of the font is cut off and it appears squished. I have found a work-around but it is clunky and needs to be done to every document that I have typed using the font.  I realize that there are probably not many people using the Plantagenet Cherokee font but I have a LOT of Word documents of things that I have transcribed or composed in my studies of the language. Has Microsoft abandoned it? Is there a fix for it? 
Thanks. 
ᏔᎻᎵ 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c0639b2c-ebde-acc8-3b34-39918c459016
* Version Independent ID: e7847622-e27d-61d8-b1f5-997ee0b925b5
* Content: [Plantagenet Cherokee - Typography](https://docs.microsoft.com/en-us/typography/font-list/plantagenet-cherokee)
* Content Source: [typographydocs/font-list/plantagenet-cherokee.md](https://github.com/MicrosoftDocs/typography/blob/live/typographydocs/font-list/plantagenet-cherokee.md)
* Product: **typography**
* GitHub Login: @alib-ms
* Microsoft Alias: **alib**

**Comment by kenmcd** [comment]
Which version of the font are you using?
MacOS comes with a version - PlantagenetCherokee.ttf, v13.0d1e3 (v1.006)
Microsoft supplemental font - plantc.ttf, v5.09 (v5.090)
And the fonts are not the same.
Is more than one version installed?

**Comment by alib-ms** [comment]
This font has not been updated for quite some time. I am not seeing any issues you have described. If you can share some details like images, sample text, I can look more into it. You may also want to try the Gadugi font shipped with Windows and Office.

**Comment by alib-ms** [comment]
This font is not abandoned. To improve our Cherokee support in Windows we have updated this typeface to the latest Unicode 14.0 published standard. In this update, we have added

-	7 missing characters from the [Cherokee U+13A0-13FF](https://unicode.org/charts/PDF/U13A0.pdf) Unicode range 

![image](https://user-images.githubusercontent.com/36778811/190723457-960efdb2-7805-43a9-a318-5a40e4257ae3.png)

-	80 new characters from [Cherokee Supplement U+AB70-ABBF](https://unicode.org/charts/PDF/UAB70.pdf) Unicode range. The Cherokee Supplement block contains lowercase letters only, whereas the Cherokee block contains all the uppercase letters, together with six lowercase letters. 
 Picture of all the newly added characters. 
 
![image](https://user-images.githubusercontent.com/36778811/190723719-12344701-1b27-4ee5-8d30-9452051fc5ac.png)
